example_bytecode_test.go

```diff
@@ -0,0 +1,107 @@
+// Example of serialization duktape function to a byte slice; deserialization from byte slice to duktape
+// function and eval in another duktape context.
+//
+// Usecase: get byte slice from duktape function (to possibly later save it to the file, send by network and etc.)
+// Please, keep in mind restrictions for a deserialized duktape function from bytecode:
+// https://github.com/svaarala/duktape/blob/master/doc/bytecode.rst
+//
+//Excerpts:
+//	When to use bytecode dump/load
+//		There are two main motivations for using bytecode dump/load:
+//			-	Performance
+//			-	Obfuscation
+
+package duktape_test
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"reflect"
+	"unsafe"
+
+	"gopkg.in/olebedev/go-duktape.v2"
+)
+
+func ExampleContext() {
```

Comment:
It seems we could give a more relevant name for this example: `func ExampleSerializeFunctuon() {`